src/Math/babylon.math.ts

```diff
@@ -4054,6 +4108,54 @@ module BABYLON {
             return new Quaternion(array[offset], array[offset + 1], array[offset + 2], array[offset + 3]);
         }
 
+        /**
+         * Create a quaternion from Euler rotation angles
+         * @param x Pitch
+         * @param y Yaw
+         * @param z Roll
+         * @returns the new Quaternion
+         */
+        public static FromEulerAngles(x: number, y: number, z: number): Quaternion {

```

Comment:
What is the value added of these functions?

Reply:
frequently i want to convert euler to quaternion and RotationYawPitchRollToRef causes confusion as parameter order is not x,y,z. Also I can never find it using autocomplete because i forget its called yawpitchroll.

Reply:
Makes sense;)